TimeTable/Models/BussinessModels/Projects/ProjectDecoder.swift

```diff
@@ -37,7 +37,7 @@ struct ProjectDecoder: Decodable {
         self.identifier = try container.decode(Int.self, forKey: .identifier)
         self.name = try container.decode(String.self, forKey: .name)
         if let colorHexString = try? container.decode(String.self, forKey: .color) {
-            self.color = UIColor(string: colorHexString)
+            self.color = UIColor(hexString: "#" + colorHexString)
```

Comment:
Is it really necessary to add `#` sign at the beginning of the hex string? It doesn't add any readability and it should work without it.